src/components/MainMenu/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+// Styles
+import { MainMenuStyled, List, ElementList, StyledLink } from './MainMenuStyles';
+
+class MainMenu extends Component {
+  render() {
+    const { menuData, match: { path } } = this.props;
+    return (
+      <MainMenuStyled>
+        <List>
+          {menuData.map((link, key) => (
+            <ElementList key={key}>
+              <StyledLink to={link.path} active={path === link.path ? 'active' : ''}>
```

Comment:
NavLink could have helped you here, you can just assign an activeClassName that will be assigned to the returned element when the route matches